core/src/test/java/org/elasticsearch/http/netty/NettyHttpRequestSizeLimitIT.java

```diff
@@ -79,11 +80,11 @@ public void testLimitsInFlightRequests() throws Exception {
             ().boundAddresses());
 
         try (NettyHttpClient nettyHttpClient = new NettyHttpClient()) {
+            @SuppressWarnings("unchecked")
```

Comment:
I think the better option is to add the '@SafeVarargs` annotation to `NettyHttpClient#post()` (which requires the method to be `final` but I'd prefer to make even the class `final` as it is not intended for inheritance).